src/controllers/dashboard/general.js

```diff
@@ -56,6 +56,12 @@ define(["jQuery", "loading", "fnchecked", "emby-checkbox", "emby-textarea", "emb
                         }
                     });
                 });
+            }, function () {
```

Comment:
don't we have to make this being called in `.then()` of inner calls, too? right now it only fires on error of `.updateServerConfiguration()` if I understand correctly

Reply:
Technically yes, but I was only adding this in response to a specific error related to the server configuration, so I didn't want to track down every other API call and add an error handler.

Reply:
So looks like we need a ticket for tracking that thing... could fix later.
Do we have a ticket for your "specific error"?

Reply:
No ticket, I just found and fixed it in like fifteen minutes.